manifests/07-operator-ibm-cloud-managed.yaml

```diff
@@ -4,6 +4,7 @@ kind: Deployment
 metadata:
   annotations:
     config.openshift.io/inject-proxy: console-operator
+    include.release.openshift.io/hypershift: "true"
```

Comment:
This file is autogenerated, thats why it should not be updated directly.
Please add the annotation instead here - https://github.com/openshift/console-operator/blob/master/profile-patches/ibm-cloud-managed/07-operator.yaml-patch

Reply:
Updated!